Misc/NEWS.d/next/Build/2023-09-17-15-17-53.gh-issue-109515.zpEECA.rst

```diff
@@ -0,0 +1 @@
+on windows, write modules to freeze into a file instead of the command line. Allow deepfreze.py to read module file names from a file with a new -f option
```

Comment:
We don't need a NEWS entry for this, it doesn't affect end users.

Reply:
done